examples/basic/src/VideoPlayer.tsx

```diff
@@ -35,6 +36,13 @@ import {AdditionalSourceInfo} from './types';
 import {bufferConfig, srcList, textTracksSelectionBy} from './constants';
 import {Overlay, toast} from './components';
 
+type AdditionnalSourceInfo = {
+  textTracks: TextTracks;
+  adTagUrl: string;
+  description: string;
+  noView: boolean;
+};
+
```

Comment:
Why was this type added if it is not used in the example?

Reply:
it has been moved in types (with a typo fix: AdditionalSourceInfo). This one can be removed !

Reply:
I removed it in following PR: https://github.com/TheWidlarzGroup/react-native-video/pull/4016